doc/api/stream.md

```diff
@@ -370,9 +370,9 @@ added: v8.0.0
 * Returns: {this}
 
 Destroy the stream. Optionally emit an `'error'` event, and emit a `'close'`
-event unless `emitClose` is set in `false`. After this call, the writable
-stream has ended and subsequent calls to `write()` or `end()` will result in
-an `ERR_STREAM_DESTROYED` error.
+event (unless `emitClose` is set to `false`) in the next tick. After this call,
```

Comment:
It is not necessarily in the next tick, it depends on the `_destroy()` implementation